src/components/aeIdentityMain/aeIdentityMain.scss

```diff
@@ -0,0 +1,125 @@
+@import './../variables';
+
+.ae-identity {
+  width: 100%;
+  zoom: 1.1;
```

Comment:
Looks like a workaround, maybe better to explicitly make font sizes and avatar bigger?